ci/test/00_setup_env_i686_centos.sh

```diff
@@ -13,3 +13,7 @@ export DOCKER_PACKAGES="gcc-c++ glibc-devel.x86_64 libstdc++-devel.x86_64 glibc-
 export GOAL="install"
 export BITCOIN_CONFIG="--enable-zmq --with-gui=qt5 --enable-reduce-exports --with-boost-process"
 export CONFIG_SHELL="/bin/dash"
+
+# gcc-c++-4.8.5 is buggy, e.g.,
```

Comment:
I'd rather wait until the next branch off so that we don't have to worry about compilers we are going to drop support for anyways.

Reply:
That's why this is a draft for now :)